spec/models/ontology_version_spec.rb

```diff
@@ -34,6 +34,26 @@
       stub_hets_for(hets_out_file('pizza'))
     end
 
+    context 'in subdirectory' do
+      let(:ontology) { create :ontology, basepath: 'subdir/pizza' }
+      let(:qualified_locid) do
+        "http://#{Settings.hostname}#{ontology_version.locid}"
+      end
+
+
+      before do
+        ontology_version
+        allow_any_instance_of(Hets::ParseCaller).to receive(:call) do |iri, *args|
+          throw(:iri, iri)
+        end
+      end
+
+      it 'should use the locid-ref for calling the parse-caller' do
```

Comment:
Seems to be a bug in hound/rubocop because the yml says:
```yml
Style/StringLiterals:
  Description: 'Checks if uses of quotes match the configured preference.'
  Enabled: true
  Style: 'single_quotes'
  EnforcedStyle: single_quotes
```